src/transformers/training_args.py

```diff
@@ -220,6 +220,10 @@ class TrainingArguments:
         seed (`int`, *optional*, defaults to 42):
             Random seed that will be set at the beginning of training. To ensure reproducibility across runs, use the
             [`~Trainer.model_init`] function to instantiate the model if it has some randomly initialized parameters.
+        data_seed (`int`, *optional*, defaults to None):
```

Comment:
```suggestion
        data_seed (`int`, *optional*):
```
Default to None is implied by optional.